dev-util/android-sdk-update-manager/android-sdk-update-manager-26.1.1.ebuild

```diff
@@ -1,82 +1,97 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI="8"
+PYTHON_COMPAT=(python3_{9..11})
 
-inherit desktop udev
-
-ANDROID_SDK_DIR="/opt/android-sdk-update-manager"
-MY_P="android-sdk_r${PV}-linux"
+inherit python-r1 udev distutils-r1
 
 DESCRIPTION="Open Handset Alliance's Android SDK"
 HOMEPAGE="https://developer.android.com"
-SRC_URI="https://dl.google.com/android/${MY_P}.tgz"
-S="${WORKDIR}/android-sdk-linux"
+SRC_URI="https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip"
+RESTRICT="mirror"
 
 LICENSE="android"
 SLOT="0"
+IUSE="udev"
 KEYWORDS="~amd64 ~x86"
-RESTRICT="mirror"
 
-DEPEND="acct-group/android"
+DEPEND="
+	acct-group/android
+	app-arch/gzip
+	app-arch/unzip
+	app-arch/tar
+"
+
 RDEPEND="
-	${DEPEND}
-	dev-java/ant-core
 	dev-java/swt:3.7[cairo]
-	>=virtual/jdk-1.8
+	>=dev-java/ant-core-1.6.5
 	sys-libs/ncurses-compat:5[abi_x86_32(-)]
 	sys-libs/zlib[abi_x86_32(-)]
+	virtual/jdk
+"
+
+ANDROID_SDK_DIR="/opt/${PN}"
+QA_FLAGS_IGNORED="
+	${ANDROID_SDK_DIR/\//}/tools/mksdcard
+	${ANDROID_SDK_DIR/\//}/tools/hprof-conv
 "
+QA_FLAGS_IGNORED_amd64="${QA_FLAGS_IGNORED}"
 
-QA_PREBUILT="*"
+QA_PREBUILT="${ANDROID_SDK_DIR/\//}/tools/*"
+
+S="${WORKDIR}/tools"
+
+src_compile() {
+	:
+}
 
 src_prepare() {
-	default
-	rm -rf tools/lib/x86* || die
+	eapply_user
 }
 
 src_install() {
-	dodoc tools/NOTICE.txt "SDK Readme.txt"
-	rm -f tools/NOTICE.txt "SDK Readme.txt" || die
+	if use udev; then
+		udev_reload
+	fi
+	dodoc "${S}/NOTICE.txt"
 
 	dodir "${ANDROID_SDK_DIR}/tools"
-	cp -pPR tools/* "${ED}${ANDROID_SDK_DIR}/tools" || die
+
+	cp -pPR "${S}"/* "${ED}${ANDROID_SDK_DIR}/tools" || die "failed to install tools"
 
 	# Maybe this is needed for the tools directory too.
 	dodir "${ANDROID_SDK_DIR}"/{add-ons,build-tools,docs,extras,platforms,platform-tools,samples,sources,system-images,temp}
-
 	fowners -R root:android "${ANDROID_SDK_DIR}"/{.,add-ons,build-tools,docs,extras,platforms,platform-tools,samples,sources,system-images,temp,tools}
 	fperms -R 0775 "${ANDROID_SDK_DIR}"/{.,add-ons,build-tools,docs,extras,platforms,platform-tools,samples,sources,system-images,temp,tools}
-
 	echo "PATH=\"${EPREFIX}${ANDROID_SDK_DIR}/tools:${EPREFIX}${ANDROID_SDK_DIR}/platform-tools\"" > "${T}/80${PN}" || die
 
 	SWT_PATH=
-	SWT_VERSIONS="3.7 3.6"
+	SWT_VERSIONS="4.10 3.8 3.7"
```

Comment:
[`3.8` is gone](https://gitweb.gentoo.org/repo/gentoo.git/commit/dev-java/swt?id=c7a97c1fec01e0f833a2c40975971853fb32651f). What about moving to `4.27`?